components/landing/TheQuickStart/PrerequisitesForWindows.vue

```diff
@@ -1,30 +1,33 @@
 <template>
   <div class="prerequisites">
     <div class="prerequisites__section">
-      <p>
-        On Windows you need to use Anaconda3 or Miniconda3 to install all the
-        dependencies.
-      </p>
-    </div>
-    <div class="prerequisites__section">
-      <h3 class="h4">
-        Install the compiler
-      </h3>
-      <pre class="code prerequisites__code">conda install --update-deps vs2017_win-64 vs2017_win-32 msvc_runtime</pre>
-    </div>
-    <div class="prerequisites__section">
-      <h3 class="h4">
-        Install OpenBLAS development headers
-      </h3>
-      <pre class="code prerequisites__code">conda install --update-deps -c conda-forge -y openblas cmake</pre>
+      <!-- eslint-disable vue/singleline-html-element-content-newline -->
+      <p><strong>Windows 64-bit:</strong> Download and run the <AppLink v-bind="rustup64Link">64-bit installer</AppLink>, which will provider follow up instructions.</p>
+      <p><strong>Windows 32-bit:</strong> Download and run the <AppLink v-bind="rustup32Link">32-bit installer</AppLink>, which will provider follow up instructions.</p>
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import { CtaClickedEventProp } from '~/constants/segment'
+
+interface TrackedLink {
```

Comment:
Same as before, we can use the `GeneralLink` interface

Reply:
Also updated w/ https://github.com/Qiskit/qiskit.org/pull/2904/commits/7c13de28f1e43a06e07c016c6b588cb144fcfa3d